### DISCUSSION TITLE
Main.css file only applying to idex file and not other pages

### DISCUSSION THREAD

**Discussion by j-sny** [twbs/bootstrap#31209] [General]
Hey I am very new to HTML and I was following a tutorial on creating a basic webpage. In the tutorial, the code `body {
background: lightgreen;
}` was used and saved as a css file in a file named styles and the reasoning explained behind this was so that it would be easier to copy and paste unto multiple pages. In the the tutorial there is a page two and a page three (which I also created) and it applied for him but not for me. It applied on the home page, so I believe the css is working on some level, but for some rease page two and three are not changing

Here's the code that I am working with 
Index

```html
<!DOCTYPE.html>
<html>
	<head>
		<meta charset="UTF-8">
		<meta name="description" content="Awesome website">
		<title>Jude's website</title>
		<link href="styles/main.css" rel="stylesheet"/>
	</head>
<body>	
	
	<header><h1>Website</h1></header>
	
	<ul>

		<li><a href="page2.html" target="_blank">Page 2</a></li>
		<li><a href="dir1/page3.html" target="_blank">Page 3</a></li>
	</ul>
</body>
</html>
```

Page 2

```html
<!DOCTYPE.html>
<html>
<head>
	<meta charset "UTF-8">
	<title>Test for link</title>
	<link href="styles/main.css" rel="stylesheet"/>
</head>
<body>
	<h1>Test for link</h1>


<a href="index.html">Home</a>

</body>
</html>
```

Page three

```html
<!DOCTYPE.html>
<html>
<head>	
	<meta charset="UTF-8">
	<title>Page 3</title>
	<link href="styles/main.css" rel="stylesheet"/>
</head>
<body>
	<h1>Page three<h1>
	

</body>
</html>
```

**Comment by abhishekrawat22** [comment]
By seeing this code everything looks good.
If CSS applied on the Home page and not on others then this can be the problem of a relative path. Make sure the address of the CSS file with respect to the page in which CSS is linked, is correct.
Also, I am seeing a 'dot(.)' applied between '<!DOCTYPE' and 'html>' for the first time.
